src/pgp.rs

```diff
@@ -289,9 +289,7 @@ pub fn dc_pgp_symm_decrypt(passphrase: &str, ctext: &[u8]) -> Option<Vec<u8>> {
 
     enc_msg
         .and_then(|msg| {
-            let mut decryptor = msg
-                .decrypt_with_password(|| passphrase.into())
-                .expect("failed decryption");
+            let mut decryptor = msg.decrypt_with_password(|| passphrase.into())?;
             decryptor.next().expect("no message")
```

Comment:
this `expect` needs to go too, otherwise malformed inputs will trigger a panic

Reply:
done.